test/fixture/validate-installed-global.js

```diff
@@ -0,0 +1,3 @@
+import test from '../../'
```

Comment:
I thought we only ignored tests/fixtures because it might have some `esnext` stuff in it, while production might not?

It would be nice if XO had an easy way to multiple configs (and a way to specify which got applied would depend on glob patterns).